bundles/org.openhab.binding.eufy/src/test/resources/state_ringing_changed.json

```diff
@@ -0,0 +1,9 @@
+{
+	"type": "event",
+	"event": {
+		"source": "device",
+		"event": "rings",
+		"serialNumber": "T8140P8121432028",
```

Comment:
I would take out / fake the serial number here because it's probably from your doorbell, better safe than sorry ;)

Reply:
whoops, seems "I would not", have to take mine out as well...